Ruby/lib/parrot.rb

```diff
@@ -20,6 +20,19 @@ def speed
     throw "Should be unreachable!"
   end
 
+  def cry
+    case @type
+    when :european_parrot
+      return "Sqoork!"
+    when :african_parrot
+      return "Sqaark!"
+    when :norwegian_blue_parrot
+      return @voltage > 0 ? "Bzzzzzz" : "..."
+    end
+
+    throw "Should be unreachable!"
```

Comment:
Normally, this would be a `raise` rather than `throw`, but I left it as `throw` to match the existing use in the previous method.

https://avdi.codes/throw-catch-raise-rescue-im-so-confused/